lib/spack/spack/build_systems/python.py

```diff
@@ -233,7 +234,22 @@ def install_args(self, spec, prefix):
         if ('py-setuptools' == spec.name or          # this is setuptools, or
             'py-setuptools' in spec._dependencies and  # it's an immediate dep
             'build' in spec._dependencies['py-setuptools'].deptypes):
-            args += ['--single-version-externally-managed', '--root=/']
+            args += ['--single-version-externally-managed']
+
+        # Get all relative paths since we set the root to `prefix`
+        pure_site_packages_dir = distutils.sysconfig.get_python_lib(
```

Comment:
This should be the distutils of the Python dependency shouldn't it?

Reply:
Oof yeah you're right. I'll run python.command.